src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/VxlanFlowSpec.groovy

```diff
@@ -249,12 +251,16 @@ class VxlanFlowSpec extends HealthCheckSpecification {
                 .build().create()
 
         then: "Flow is created with protected path"
-        def flowPathInfo = flow.retrieveDetails()
-        flowPathInfo.allocateProtectedPath
+        def flowPathInfo = flow.retrieveAllEntityPaths()
```

Comment:
Oh, it is insane, it was me who wrote `def flowPathInfo = flow.retrieveDetails()`. Sorry for this typo :)

Reply:
np 😉